cloud-driver-libs/cloudos_meta.yaml

```
@@ -13,6 +13,7 @@ AWS:
 GCP:
   region: Region Zone
   credential: PrivateKey ProjectID ClientEmail
+  rookdisktype: pd-standard/pd-balanced/pd-ssd/pd-extreme
```

이렇게 적는 것은 어떨지 제안해 봅니다 😊

```suggestion
  rookdisktype: 
    - pd-standard
    - pd-balanced
    - pd-ssd
    - pd-extreme
```

---

@jihoon-seo 

- 순서가 보장되나요?

---

예를 들어
https://github.com/cloud-barista/cb-spider/blob/master/.github/workflows/prow-commands.yml 를 보면
`steps:` 아래의
`- name: Check author_association` 와 `- name: Action according to command` 가 순서대로 실행되는 것을 봐서
YAML 내의 unordered list 도 나열순서대로 읽을 수 있는 것 같구요

잠깐 찾아 보니

- https://github.com/go-yaml/yaml/issues/30
(CB-Spider의 `go.mod`를 보니 
`gopkg.in/yaml.v2` 와 `gopkg.in/yaml.v3` 을 활용하고 있네요)
- https://stackoverflow.com/questions/33639269/preserving-order-of-yaml-maps-using-go/42109240
- https://issueexplorer.com/issue/google/go-jsonnet/222

이런 것들이 나오는데
이 경우에 맞는 페이지들인지는 잘 모르겠습니다.

---

만약 위와 같이 바꾼다면
순서가 보장되는지 테스트가 필요할 것 같습니다. 😊

---

- 캄사~
- 구현시 참고해서 좀더 고민해보겠습니다.